### DISCUSSION TITLE
Allow Mastodon registration only from selected instances

### DISCUSSION THREAD

**Discussion by D1reW0lf** [pixelfed/pixelfed#4751] [Ideas]
Since "Allow registration from Mastodon" bypasses the "Registration closed" when PF_LOGIN_WITH_MASTODON_ENABLED_SKIP_CLOSED is active, would it be possible to use the PF_LOGIN_WITH_MASTODON_DOMAINS list as the only accepted instances?

**Comment by dansup** [comment]
Yes, with `PF_LOGIN_WITH_MASTODON_DOMAINS=mastodon.social,mastodon.art` and `PF_LOGIN_WITH_MASTODON_ONLY_CUSTOM=true` you could limit to `mastodon.social` and `mastodon.art` only!
